programs/ssl/ssl_client2.c

```diff
@@ -2361,25 +2362,25 @@ int main( int argc, char *argv[] )
             goto exit;
         }
 
-        if( ( ret = mbedtls_ssl_conf_psk_opaque( &conf, slot,
+        if( ( ssl_conf_ret = mbedtls_ssl_conf_psk_opaque( &conf, slot,
```

Comment:
`ssl_conf_ret` is used only locally here thus I don't see the point. Isn't it a leftover from previous changes? Furthermore with this we don't see the message "Last error was;" ...